deps/build.jl

```diff
@@ -34,6 +34,10 @@ if is_apple()
     provides(Homebrew.HB, "homebrew/science/hdf5", hdf5, os=:Darwin)
 end
 
+if Sys.KERNEL === :FreeBSD
```

Comment:
Actually, do we really need the `if` statement here... I think this can just be put in the `is_linux()` branch and let `BinDeps` find a provider.